src/content/docs/guides/node-setup/deploy-a-node/index.mdx

```diff
@@ -125,15 +125,27 @@ rpm -i tenzir-static-amd64-linux.rpm
 </TabItem>
 <TabItem value="nix" label="Nix" icon="nix">
 
-Use our `flake.nix`:
+Use our `flake.nix` to run an ad-hoc tenzir process:
 
 ```bash
-nix run github:tenzir/tenzir/stable
+nix run github:tenzir/tenzir/latest
 ```
 
-Install Tenzir by adding `github:tenzir/tenzir/stable` to your flake inputs, or
+Or run a tenzir node without installing:
+
+```bash
+nix shell github:tenzir/tenzir/latest -c tenzir-node
+```
+
+Install Tenzir by adding `github:tenzir/tenzir/latest` to your flake inputs, or
```

Comment:
We have two products: Tenzir Node and Tenzir Platform. You cannot install "Tenzir".